homeassistant/components/smartthings/sensor.py

```diff
@@ -116,6 +116,9 @@
             None,
         )
     ],
+    Capability.gas_meter: [
+        Map(Attribute.gas_meter, "Gas Meter", ENERGY_KILO_WATT_HOUR, None)
```

Comment:
Shouldn't we add all of the attributes in the capability?  They may not be populated for your device, but it may for others':

```json
    "gasMeter": [
        "gasMeter",
        "gasMeterCalorific",
        "gasMeterConversion",
        "gasMeterPrecision",
        "gasMeterTime",
        "gasMeterVolume",
    ]
```

Reply:
Hmm, I

Reply:
Hmm, I could give it a shot, but I don't know if it's useful to import all of them. As far as I'm aware this is only supported by Bulb's latest generated of energy meters, and that has only been giving me data for gasMeter. Additionally, quite a few of the attributes are intended to be used more like static values (precision/conversion) which may not make sense to expose as a sensor?

Reply:
This revealed I screwed up the original PR. https://github.com/andrewsayre/pysmartthings/pull/43 to fix.